src/Framework/TestCase.php

```diff
@@ -1007,6 +1014,11 @@ final protected function expectOutputString(string $expectedString): void
         $this->outputExpectedString = $expectedString;
     }
 
+    final protected function expectsErrorLog(): void
```

Comment:
This should be `expectErrorLog()` to be consistent with `expectOutputString`, for example.